common/src/main/kotlin/streams/configuration/StreamsConfigProcedures.kt

```diff
@@ -0,0 +1,50 @@
+package streams.configuration
+
+import org.neo4j.graphdb.GraphDatabaseService
+import org.neo4j.kernel.internal.GraphDatabaseAPI
+import org.neo4j.logging.Log
+import org.neo4j.procedure.Context
+import org.neo4j.procedure.Description
+import org.neo4j.procedure.Mode
+import org.neo4j.procedure.Name
+import org.neo4j.procedure.Procedure
+import streams.events.KeyValueResult
+import java.util.stream.Stream
+
+class StreamsConfigProcedures {
+
+    @JvmField @Context
+    var log: Log? = null
+
+    @JvmField @Context
+    var db: GraphDatabaseService? = null
+
+    @Procedure
```

Comment:
Security -- need admin only annotation, or otherwise, we need to add docs indicating how people can restrict the usage of this procedure.  The config may include secret tokens & keys to auth to kafka that regular users should not be able to obtain.

Reply:
Done